code/game/gamemodes/wizard/artefact.dm

```diff
@@ -82,6 +82,55 @@
 	rend_desc = "Gently wafting with the sounds of endless laughter."
 	icon_state = "clownrender"
 
+
+//--- BECAUSE IT'S FUCKING HILARIOUS --//
+/obj/item/weapon/veilrender/uncharged
+	name = "wicked blade"
+	desc = "A replica of a notorious eldritch artifact of immense power. Legend tells that those who capture two powerful souls with it will weild the weapon of legends."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "render"
+	item_state = "render"
+	force = 5 //shitty wizard build quality
+	throwforce = 0
+	w_class = 3
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	charges = 1
+	spawn_type = /obj/singularity/narsie/wizard
+	spawn_amt = 1
+	activate_descriptor = "the blood geometer's prison"
+	rend_desc = "You hear the sound of laughing slaughter demons."
+	spawn_fast = 1
+	var/captured_ascendants = 0
+
+/obj/item/weapon/veilrender/uncharged/attack_self(mob/user)
+	if(charges > 0 && captured_ascendants==2)
+		new /obj/effect/rend(get_turf(user), spawn_type, spawn_amt, rend_desc, spawn_fast)
+		charges--
+		captured_ascendants = 0
+		user << "<span class='boldannounce'>You merge the two ascendants' souls and drive them into bluespace. You hear blood curdling screams emitting from the souls and the hole to bluespace opens wider. Uh oh.</span>"//Blood dimension = bluespace because EMERGENT LORE EXPANSION is sorely needed
+		user.visible_message("<span class='boldannounce'>[src] hums with power as [user] deals a blow to [activate_descriptor] itself!</span>")
+		world << "span class='shadowling'>NAR`SIE  WGHA'N SHU-AIIIIEEEEEEEEEEE!!!!</span>"//Those ascendants are healthy snacks to make sure your interdinmensional horror grows up big and strong!
+	else
+		user << "<span class='danger'>The blade lies quiet.</span>"
+
+/obj/item/weapon/veilrender/uncharged/attack(mob/living/M, mob/living/user)
+	if(!iswizard(user))
+		user.Paralyse(5)
+		user.adjustBrainLoss(5)//Aay it's an eldritch artifact, I ain't gotta explain shit!
+		user << "<span class='userdanger'>Your mind oozes as you try to figure out how to use the sanity-shattering blade!</span>"
+		return ..()
+	if(!istype(M, /mob/living/simple_animal/ascendant_shadowling))
+		return ..()

```

Comment:
Perhaps we should move the iswizard check AFTER this, so that muggles can still stab humans with the magical knife.